pages-resources/resources-links.html

```diff
@@ -0,0 +1,71 @@
+<div class="row">
+    <div class="col-md-12">
+        <a name="resources-ides"></a>
+        <h2>
+            Useful Links
+        </h2>
+    </div>
+</div>
+<div class="row">
+    <div class="col-2 resource-box">
+        <div class="text-center resource-icon">
+            <a class="color-github" href="https://github.com/CodeWithGroup/" target="_blank">
+                <span class="fab fa-4x fa-github"></span>
+            </a>
+        </div>
+    </div>
+    <div class="col-10 resource-description">
+        <h4>
+            <a href="https://github.com/CodeWithGroup/" target="_blank">GitHub</a>
+        </h4>
+        <p>
+            Git is industry-leading version control software and is incredibly popular with professionals and hobbyists
+            alike. CodeWith has a public GitHub repository where we encourage you to store any public projects you work
+            on at
+            CodeUp. We even run this web site direct from our GitHub repository.
+        </p>
+
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-2 resource-box">
+        <div class="text-center resource-icon">
+            <a class="color-slack"
+                href="https://join.slack.com/t/codeupmanchester/shared_invite/enQtMzI4OTYyOTg1NTU1LTAxODBiMzg1YjMxMzhjYmRlN2VjMmE1YTIwYmI2ZDQxZmZhODlhZTVlZWU4OTY3YTU0ZDE2YjQzNDgyMjRiYTU"
+                target="_blank">
+                <span class="fab fa-4x fa-slack"></span>
+            </a>
+        </div>
+    </div>
+    <div class="col-md-10 resource-description">
+        <h4>
+            <a href="https://launchpass.com/codewithpeople" target="_blank">Slack</a>
+        </h4>
+        <p>
+            CodeWith has a Slack workspace where you can chat, share files, ask questions and get help with problems in
+            between Meetups. Follow the link to get an invitation.
+        </p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-2 resource-box">
+        <div class="text-center resource-icon">
+            <a class="color-slides"
+                href="https://join.slack.com/t/codeupmanchester/shared_invite/enQtMzI4OTYyOTg1NTU1LTAxODBiMzg1YjMxMzhjYmRlN2VjMmE1YTIwYmI2ZDQxZmZhODlhZTVlZWU4OTY3YTU0ZDE2YjQzNDgyMjRiYTU"
```

Comment:
This is the same link (to Slack) as above, not a Google Drive link

Reply:
It's linked twice, fixed now